handbook/openid-connect/openid-connect.md

```diff
@@ -0,0 +1,223 @@
+# Single Sign-on with OpenID Connect
+
+You may configure SystemLink to use [OpenID Connect](https://openid.net) for user authorization. This enables SystemLink to leverage corporate single sign-on (SSO) and the additional security benefits it provides such streamlined login and limiting the proliferation of user credentials. This also enables SystemLink to utilize a common identity for users across multiple applications. OpenID Connect can be used in conjunction with or as a replacement for LDAP, Active Directory, and local Windows accounts for authentication. 
+
+
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2020R4 or greater. Please see [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server. 
+- A DNS name for the SystemLink server. 
+- SystemLink login with the **Server Administrator** role. 
+- Administrator desktop access to the SystemLink server
+- A OpenID Connect Provider server such as [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html), [Azure ADFS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs), [Okta](https://www.okta.com/openid-connect/), or another [certified provider](https://openid.net/certification/) that has been fully setup and configured for OpenID Connect authentication. 
+    - If you have not yet setup your provider please consult the vendor documentation for setup and configuration. 
+
+## Enabling OpenID Connect in SystemLink
+1. Log into the server running SystemLink and open **NI Web Server Configuration**
+2. Go to the **Authentication** tab and enable both **Log in as users controlled by the web server** and **Use OpenID Connect (advanced)**
+3. Add the configuration files to SystemLink to connect to your OpenID Connect provider and confirm that you can log in as an openid connect user
+> **Note:** details on this process is found in [**OpenID Configuration Files in SystemLink Server**](#openid-configuration-files-in-systemlink-server)
```

Comment:
These look a little funny, the Note text is lighter than the other content, except the link is still dark. Maybe a we can tweak the css a bit.

![image](https://user-images.githubusercontent.com/62968881/98415108-3be92e80-2042-11eb-8be1-22940135e28d.png)

Reply:
This is addressed by using admonitions here rather than block quotes.